kcidb/templates/stable_rt.j2

```diff
@@ -0,0 +1,3 @@
+{# stable-rc macros #}
+{% set selected_origins = ['maestro', 'broonie'] %}
+{% set indent = "      " %}
```

Comment:
I wonder if you could merge stable_rc.j2 and stable_rt.j2 into a single stable.j2?

Reply:
Are there any other ways you could extract code that's shared between these two subscriptions?

Reply:
Another way is to use context variables while rendering the templates here https://github.com/kernelci/kcidb/blob/main/kcidb/monitor/output.py#L196.
But creating templates and importing them looks much simpler to me than tracking down context variables.

Reply:
Created `common.j2` to share context variables among `stable-rc` and `stable-rt` report templates.